packages/navcookiebanner/src/react/index.js

```diff
@@ -0,0 +1,80 @@
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+import { CloseIcon } from '@pluralsight/ps-design-system-icon'
+import { compose, css, media } from 'glamor'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useCookies } from 'react-cookie'
+
+import stylesheet from '../css/index.js'
+
+const DISMISS_COOKIE_NAME = 'prism-cookienotify'
+const DEFAULT_PRIVACY_UPDATED_DATE = new Date('2018-05-24T00:00:00')
+
+const styles = {
+  banner: () =>
+    compose(
+      css(stylesheet['.psds-navcookiebanner']),
+      media(
+        '(min-width: 769px)',
+        css(stylesheet['@media (min-width: 769px)']['.psds-navcookiebanner'])
+      ),
+      media('print', css(stylesheet['@media print']['.psds-navcookiebanner']))
+    ),
+  message: () => css(stylesheet['.psds-navcookiebanner__message']),
+  dismiss: () => css(stylesheet['.psds-navcookiebanner__dismiss'])
+}
+
+const NavCookieBanner = React.forwardRef((props, forwardedRef) => {
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
+  const [cookies, setCookie] = useCookies(['cookie-name'])
```

Comment:
what is `cookie-name` used for here

Reply:
in the src - the array is for dependencies for when to update. i'm guessing we can just pass it nothing?

Reply:
It's supposed to be the name of the cookie I'm setting so that state updates can be detected.  Thanks for catching this.  This lets me get rid of the secondary state update for `clicked`.  This was my misuse of the lib.